.github/workflows/compliance.yml

```diff
@@ -17,6 +17,7 @@ jobs:
       - uses: mtfoley/pr-compliance-action@main
         with:
           body-auto-close: false
+          body-regex: (fix|resolv|clos|r)(e)*(s|d)* (#[1-9]+\d*|\S+\/\S+\/issues\/[1-9]+\d*)
```

Comment:
> allow linking to external issues

I'm a -1 on that, personally. Normally I'd want people to file an issue on _this_ repo. You (@RebeccaStevens) are a bit privileged in that I know you're doing great stuff - but otherwise for the most part I wouldn't want newcomers to do what you're doing.

Reply:
Do you want me to just change this PR to only disallowing leading zeros for now?
Or do you want to wait on the upstream issue?

Reply:
Change made if you want to go with the former.

I also made the word matching better.

Reply:
Hmm yeah I'd rather it just land upstream. It's not a common case.